tests/com.b2international.snowowl.test.dependencies/.classpath

```diff
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry exported="true" kind="lib" path="lib/annotations-17.0.0.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/commons-compress-1.21.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/docker-java-api-3.2.13.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/docker-java-transport-3.2.13.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/docker-java-transport-zerodep-3.2.13.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/duct-tape-1.0.8.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/elasticsearch-1.17.3.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/jna-5.8.0.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/system-rules-1.18.0-sources.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/testcontainers-1.17.3.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/system-rules-1.18.0-sources.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/system-rules-1.18.0.jar" sourcepath="lib/system-rules-1.18.0-sources.jar"/>
```

Comment:
The first entry should not be added as a library, as now it is properly set as a source library.